.github/workflows/rebase.yml

```diff
@@ -1,18 +1,20 @@
+name: Automatic Rebase
+
 on:
   issue_comment:
-    types: [created]
-name: Automatic Rebase
+    types: [ created ]
+
 jobs:
   rebase:
     name: Rebase
     if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase')
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the latest code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
       - name: Automatic Rebase
-        uses: cirrus-actions/rebase@1.5
+        uses: cirrus-actions/rebase@v1
```

Comment:
Why do use `v1` instead of `v1.5`?

Reply:
`v1` will use the latest version.

Reply:
[The version 1.5]( https://github.com/cirrus-actions/rebase/releases/tag/1.5) is latest, is not it?

Reply:
For the later days, no need to bump the minor versions.

Reply:
I would like to use `v1.5` that has some fixes. Also dependbot can force to use 1.5 https://github.com/Kotlin/dokka/pull/1920

Reply:
ecb4cefe40941cc0f6507eee9664c6d5840d02a1

Reply:
Yep

Reply:
2604544b2fc65e6472ea1139d698109ef8cd37e2

Reply:
Thank you